From: Ryan Lee <ryan.lee@canonical.com>
Date: Wed, 12 Feb 2025 09:42:55 -0800
Subject: [PATCH] delete the busybox and nautilus profiles

Both those profiles are for programs that are installed by default and
that provide or open a shell (which can then be used to perform arbitrary
actions, thus providing a bypass for the unprivileged usernamespace
restrictions.

Signed-off-by: Ryan Lee <ryan.lee@canonical.com>
---
 profiles/apparmor.d/busybox  | 13 -------------
 profiles/apparmor.d/nautilus | 13 -------------
 2 files changed, 26 deletions(-)
 delete mode 100644 profiles/apparmor.d/busybox
 delete mode 100644 profiles/apparmor.d/nautilus

diff --git a/profiles/apparmor.d/busybox b/profiles/apparmor.d/busybox
deleted file mode 100644
index 23ba942..0000000
--- a/profiles/apparmor.d/busybox
+++ /dev/null
@@ -1,13 +0,0 @@
-# This profile allows everything and only exists to give the
-# application a name instead of having the label "unconfined"
-
-abi <abi/5.0>,
-include <tunables/global>
-
-profile busybox /usr/bin/busybox flags=(unconfined) {
-  userns,
-  @{exec_path} mr,
-
-  # Site-specific additions and overrides. See local/README for details.
-  include if exists <local/busybox>
-}
diff --git a/profiles/apparmor.d/nautilus b/profiles/apparmor.d/nautilus
deleted file mode 100644
index 4024989..0000000
--- a/profiles/apparmor.d/nautilus
+++ /dev/null
@@ -1,13 +0,0 @@
-# This profile allows everything and only exists to give the
-# application a name instead of having the label "unconfined"
-
-abi <abi/5.0>,
-include <tunables/global>
-
-profile nautilus /usr/bin/nautilus flags=(unconfined) {
-  userns,
-  @{exec_path} mr,
-
-  # Site-specific additions and overrides.  See local/README for details.
-  include if exists <local/nautilus>
-}
